### PR TITLE
DAOS-13017 doc: Document checkpointing properties

### PR DESCRIPTION
**Pull request by jolivier23**
Add documentation for checkpoint control properties.

Doc-only: True
Required-githooks: true

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

**Review by NiuYawei — APPROVED** [review]

**Comment by jolivier23** [comment]
ping

File: docs/admin/pool_operations.md [hunk 1]
```diff
@@ -392,6 +392,35 @@ to tolerate. Valid values are between 0 to 4, inclusive, with 2 being the
...
+three supported options:
+
+* "timed"       : Checkpointing is also triggered periodically (default option).
+* "lazy"        : Checkpointing is only triggered when there is WAL space pressure.
+* "disabled"    : Checkpointing is disabled.  WAL space may be exhausted.
+
+#### Checkpoint frequency (checkpoint\_freq)
+
+This property controls how often checkpoints are triggered.  If the checkpoint policy is
+not "timed", this property is ignored. The value is specified in seconds.
```
Comment (Michael-Hennecke) [inline docs/admin/pool_operations.md:416]: min/max/default value?

File: docs/admin/pool_operations.md [hunk 2]
```diff
@@ -392,6 +392,35 @@ to tolerate. Valid values are between 0 to 4, inclusive, with 2 being the
...
+
+#### Checkpoint threshold (checkpoint\_thresh)
+
+This property controls the percentage of WAL usage to automatically trigger a checkpoint.
+The value is specified as a percentage between 10-75.  Anything outside the range will be
+automatically adjusted.
+
 See [Erasure Code](https://docs.daos.io/v2.4/user/container/#erasure-code) for details on
 erasure coding at the container level.
```
Comment (Michael-Hennecke) [inline docs/admin/pool_operations.md:425]: this paragraph look s mis-placed after the inserted checkpoint documentation ... maybe move it up into the "Default EC size" section?

**Review by Michael-Hennecke — COMMENTED** [review]
minor comments inline

**Review by gnailzenh — APPROVED** [review]

**Review by NiuYawei — APPROVED** [review]

**Pull request merged**